src/plugins/cspell/index.ts

```diff
@@ -0,0 +1,40 @@
+import { _load } from '../../util/loader.js';
+import { timerify } from '../../util/performance.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { PluginConfig } from './types.js';
+import type { IsPluginEnabledCallback, GenericPluginCallback } from '../../types/plugins.js';
+
+// link to docs
+
+export const NAME = 'cspell';
+
+/** @public */
+export const ENABLERS = ['cspell'];
+
+export const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(dependencies, ENABLERS);
+
+// https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#customization
```

Comment:
Sorry for the nittiest of nitpicks to pick, but all plugins have a link to the docs up there at "link to docs"